administration/src/util/helper.ts

```diff
@@ -13,3 +13,10 @@ export type UnionToIntersection<U> = (U extends any ? (x: U) => void : never) ex
 
 const multipleSpacePattern = /\s\s+/g
 export const removeMultipleSpaces = (value: string): string => value.replace(multipleSpacePattern, ' ')
+export const containsSpecialCharacters = (value: string): boolean =>
+  /[`!@#$%^&*()_+\-=\]{};':"\\|,.<>?~0123456789]/.test(value)
+
+export const containsNoEmojis = (value: string): boolean =>
```

Comment:
:wrench: I think it would be nice if the function is not negated. "containsEmojis" to avoid double negation when being used and being more consistent with "containsSpecialChars"